### ISSUE TITLE
SSL Expired

### ISSUE THREAD

**Issue by sggr57a** [knazarov/homebrew-qemu-virgl#61]
Got this error upon git clone...

fatal: unable to access 'https://gitlab.freedesktop.org/virgl/virglrenderer.git/': SSL certificate problem: certificate has expired

**Comment by knazarov** [comment]
Yeah, this likely was an issue of gitlab on the freedesktop site. This seems to be fixed.

**Comment by DUOLabs333** [comment]
This should be closed now.
